src/components/highlighter/Highlighter.js

```diff
@@ -1,31 +0,0 @@
-import React from "react";
-import Highlight from "react-highlighter";
-import PropTypes from "prop-types";
-import { withStyles } from "@material-ui/core/styles/index";
-
-/**
- * @author aramsey
- * A function which will take a search term and some text,
- * and highlight the search term within the text
- */
```

Comment:
I thought we intended to keep this component for search results?
It's also marked as `Keep` on our clean-up spreadsheet.

Reply:
Oops! Yes. My Bad. Will restore them.